pom.xml

```diff
@@ -13,6 +13,12 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <jenkins-test-harness.version>1.625</jenkins-test-harness.version>
```

Comment:
Yes, JTH and core versions are usually aligned andin my experience with the PCT you need to keep both coherent, also without the JTH bump the build fails due to banned dependencies (the 1.424 JTH includes some old versions not compatible with the new enforcer rules)